package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graffy-project",
-  "version": "0.0.2",
+  "version": "0.16.0",
```

Comment:
What! We never changed the version in `package.json` 🤣

Reply:
Haha yeah. The version in this root package.json doesn't really matter. The build script generates package.json files for the published packages. The version in those come from the logic in scripts/version.js.